### PR TITLE
Fixes:Event End time can be earlier than Start time

### LINKED ISSUE #1901: Event End time can be earlier than Start time
**Describe the bug**
user can create event with end time earlier than start time.

**To Reproduce**

1.Go to /orgevents/id={id} in talawa-admin
2.Click on Add Event
3.Turn off the all day option
4.now you can create event with end time before the start time

**Expected behavior**
API should throw an error message saying "end time shouldn't be earler than start time"

**Actual behavior**
user can set the time backwards  and the api registers it  

**Screenshots**

Screenshot from 2024-02-27 19-58-48

**Additional details**
related to [#1618](https://github.com/PalisadoesFoundation/talawa-admin/issues/1618)
@Cioppolo14  Please assign me this as I'm working on this on the admin side already.

**Potential internship candidates**
Please read this if you are planning to apply for a Palisadoes Foundation internship https://github.com/PalisadoesFoundation/talawa/issues/359

### PR DESCRIPTION
**Pull request by wasifkareem**
**What kind of change does this PR introduce?**
This PR solve the events time issue

**Issue Number:**

Fixes #1901 

**Did you add tests for your changes?**
Yes

**Snapshots/Videos:**
Screenshot from 2024-03-03 00-48-07

**If relevant, did you update the documentation?**

**Summary**
Now if user tries to put event end time earlier than start time the api will throw an error saying "end time can't be earlier than start time"

**Does this PR introduce a breaking change?**

**Other information**

**Have you read the [contributing guide](https://github.com/PalisadoesFoundation/talawa-api/blob/master/CONTRIBUTING.md)?**

### CODE REVIEW

**Comment by wasifkareem** [comment]
Hi @palisadoes , My first ever PR is ready for review

**Comment by palisadoes** [comment]
Please fix the failing tests. Linting.

**Comment by wasifkareem** [comment]
> Please fix the failing tests. Linting.

"All tests have been passed successfully." @palisadoes , please review

File: tests/resolvers/Mutation/createEvent.spec.ts [hunk 1]
```diff
@@ -1221,27 +1221,19 @@ describe("Check for validation conditions", () => {
...
-          startDate: new Date().toUTCString(),
-          startTime: new Date().toUTCString(),
-          title: "newTitle",
+          startDate: "",
+          startTime: "2024-03-02T08:00:00.000Z",
+          title: "Random",
           recurrance: "DAILY",
-          images: [
-            "image_url_1.jpg",
-            "image_url_2.jpg",
```
Comment (palisadoes) [inline tests/resolvers/Mutation/createEvent.spec.ts:1239]: 1. In the event test, you have removed image upload testing which was a valid test. A PR was recently merged to add and test that functionality.
2. Please add a new test for this bug

**Review by palisadoes — CHANGES_REQUESTED** [review]
See comment

File: tests/resolvers/Mutation/createEvent.spec.ts (hunk 1 above)
Comment (wasifkareem) [reply tests/resolvers/Mutation/createEvent.spec.ts]: Hi @palisadoes , I re-added the tests for image uploads, please have a look

**Comment by wasifkareem** [comment]
Hi @palisadoes , @sumitra19jha , @DMills27 Can you please review this PR so that I can close this issue, I want to contribute to more issues, but I am limited by the two-issue policy.

**Review by palisadoes — APPROVED** [review]

**Pull request merged**